Library/Formula/racket.rb

```diff
@@ -28,17 +31,43 @@ def install
       system "make"
       system "make", "install"
     end
+
+    # configure racket's package tool (raco) to do the Right Thing
+    # see: http://docs.racket-lang.org/raco/config-file.html
+    inreplace etc/"racket/config.rktd" do |s|
+        s.gsub!(
+            /\(bin-dir\s+\.\s+"#{Regexp.quote(bin)}"\)/,
```

Comment:
The result of `Regexp.quote` is variable because its input is variable; it can't be hard-coded.